group_vars/zookeeper/staging.yml

```diff
@@ -7,4 +7,3 @@ zk_nodes:
     id: 2
   - host: "lib-zk-staging6.princeton.edu"
```

Comment:
Here we use the same exact variable as above. I see Future Me updating it in one place and forgetting about the other. Can we consolidate?

Reply:
Oh weird. I'm seeing lib-zk-staging{4,5,6}.princeton.edu with id's {1,2,3} respectively.